tests/unit/test_charm.py

```diff
@@ -4,72 +4,89 @@
 # Learn more about testing at: https://juju.is/docs/sdk/testing
 
 import unittest
+from unittest.mock import Mock, PropertyMock, patch
 
 import ops.testing
-from ops.model import ActiveStatus, BlockedStatus, WaitingStatus
+from ops.model import ActiveStatus, BlockedStatus, Container, WaitingStatus
+from ops.pebble import Error as PebbleError
+from ops.pebble import PathError
 from ops.testing import Harness
 
 from charm import MimirK8SOperatorCharm
 
 
 class TestCharm(unittest.TestCase):
```

Comment:
I wonder if should move away from `test_charm` and `TestCharm` in favor of more scoped naming, for example:
- `test_pebble_layer.py`
  - `class TestPebbleLayer`
    - `def test_in_isolation()`
      - assert service is running
      - assert pebble command args
    - `def test_with_external_url()`
      - assert arg has external url
- `test_charm_status.py`
  - `class TestPebbleErrors`
    - `def test_pebble_error_on_push_blocks_charm`

Reply:
I did not do it because the charm is now in its first steps, we can do ir though